api/grpcserver.go

```diff
@@ -401,17 +401,6 @@ func (svr *gRPCHandler) EstimateActionGasConsumption(ctx context.Context, in *io
 		if err := sc.LoadProto(in.GetExecution()); err != nil {
 			return nil, status.Error(codes.InvalidArgument, err.Error())
 		}
-		var (
-			gasPrice *big.Int = big.NewInt(0)
-			ok       bool
-		)
-		if in.GetGasPrice() != "" {
-			gasPrice, ok = big.NewInt(0).SetString(in.GetGasPrice(), 10)
-			if !ok {
-				return nil, status.Error(codes.InvalidArgument, "invalid gas price")
-			}
-		}
-		sc.SetGasPrice(gasPrice)
 		ret, err := svr.coreService.EstimateExecutionGasConsumption(ctx, sc, callerAddr)
```

Comment:
IMO, should keep same gasprice logic between grpc and web3.

Reply:
also may not be necessary to delete this logic, since the gas price has already been changed to 0 in coreservice.

Reply:
updated